libraries/core/src/main/java/com/paritytrading/nassau/soupbintcp/SoupBinTCP.java

```diff
@@ -42,17 +42,24 @@ private SoupBinTCP() {
      * Payload for the Login Accepted packet.
      */
     public static class LoginAccepted {
-        public byte[] session;
-        public byte[] sequenceNumber;
+        private byte[] session;
+        private byte[] sequenceNumber;
 
```

Comment:
Encapsulate the underlying buffers via constructor initialization and public get methods.